docs/specification.md

```diff
@@ -221,8 +236,6 @@ The list of erros may be inaccurate and incomplete, it will be updated during th
 | `NOT_TOKEN_OWNER`               | Trying to configure operators for a different wallet which sender does not own                              |
 | `FAIL_PROPOSAL_CHECK`           | Throws when trying to propose a proposal that does not pass `proposalCheck`                                 |
 | `FROZEN_TOKEN_NOT_TRANSFERABLE` | Transfer entrypoint is called for frozen token by a non-admin sender                                        |
-| `PROPOSAL_INSUFFICIENT_BALANCE` | Throws when trying to propose a proposal without having enough unfrozen token                               |
-| `VOTING_INSUFFICIENT_BALANCE`   | Throws when trying to vote on a proposal without having enough unfrozen token                               |
```

Comment:
It looks like we still have error definitions for both of these in the `Ligo.BaseDAO.Types` module, can you remove them?

Reply:
Removed.